server/schedule/operator/step.go

```diff
@@ -82,6 +82,70 @@ func (tl TransferLeader) Influence(opInfluence OpInfluence, region *core.RegionI
 	to.LeaderCount++
 }
 
+// EvictLeader is an OpStep that transfers a region's leader to one of the targets.
+type EvictLeader struct {
+	FromStore uint64
+	ToStore   uint64
```

Comment:
Why do we still need it?

Reply:
For compatible, [ref](https://github.com/tikv/pd/pull/4214#issuecomment-947587897).

Reply:
I think we don't need this field here, it can be appended when we send it to TiKV?